experimental/framework/azure-client/README.md

```diff
@@ -90,7 +90,7 @@ const schema = {
     },
     dynamicObjectTypes: [ /*...*/ ],
 }
-const frsClient = new FrsClient(config);
+const frsClient = new AzureClient(config);
```

Comment:
Do we want to change the variable name (frsClient) here

Reply:
Updated!